string SceneName
geometry_msgs/Pose StartPoint
geometry_msgs/Pose GoalPoint